src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -93,29 +94,20 @@ const defaultProps: Partial<ItemListLayoutProps> = {
   customizeTableRowProps: () => ({} as TableRowProps),
 };
 
-interface ItemListLayoutUserSettings {
-  showAppliedFilters?: boolean;
-}
+const localStorage = createStorage("item_list_layout", {
+  showFilters: false, // setup defaults
+});
 
 @observer
 export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   static defaultProps = defaultProps as object;
+  
+  get showFilters(): boolean {
+    return localStorage.get().showFilters;
+  }
 
-  @observable userSettings: ItemListLayoutUserSettings = {
-    showAppliedFilters: false,
-  };
-
-  constructor(props: ItemListLayoutProps) {
-    super(props);
-
-    // keep ui user settings in local storage
-    const defaultUserSettings = toJS(this.userSettings);
-    const storage = createStorage<ItemListLayoutUserSettings>("items_list_layout", defaultUserSettings);
-
-    Object.assign(this.userSettings, storage.get()); // restore
-    disposeOnUnmount(this, [
-      reaction(() => toJS(this.userSettings), settings => storage.set(settings)),
-    ]);
+  set showFilters(showFilters: boolean) {
+    localStorage.merge({ showFilters });
```

Comment:
These are changes to public api, right?. We cannot make these in a patch release.

Reply:
Yes, we can. It's component internals (not input props). It matter only if you use `ref={}`, but it's not even documented and I doubt used anywhere..